### PR TITLE
Add multi-zone AHU entity type

### PR DESCRIPTION
**Pull request by briancjohns**
Added AHU_SFSS_SFVSC_EFSS_CHWSC_RTM_FDPM2X_SSPC_OAFC_HTSC

### CODE REVIEW

File: ontology/yaml/resources/HVAC/entity_types/AHU.yaml [hunk 1]
```diff
@@ -5317,3 +5317,19 @@ AHU_BSPC_DX4SC_ECON_EFSS_EFVSC_SFSS_SFVSC_SSPC_RHHC_NONCANONICAL_1:
...
+  guid: "75559dbd-7ef6-4368-b48f-8d34db7fe47d"
+  description: "Multi zone AHU with exhaust fan, two prefilters, outside air, electric supply side heating, and supply static pressure control"
+  is_canonical: true
+  implements:
+  - AHU
+  - SFSS
+  - EFSS
+  - SFVSC
+  - CHWSC
+  - RTM
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/AHU.yaml:5331]: remove RTM (return temp will be covered with the addition of ECON)

File: ontology/yaml/resources/HVAC/entity_types/AHU.yaml [hunk 2]
```diff
@@ -5317,3 +5317,19 @@ AHU_BSPC_DX4SC_ECON_EFSS_EFVSC_SFSS_SFVSC_SSPC_RHHC_NONCANONICAL_1:
   - heating_water_valve_percentage_command
   - min_zone_air_relative_humidity_sensor
   - min_zone_air_relative_humidity_setpoint
+
+AHU_SFSS_SFVSC_EFSS_CHWSC_RTM_FDPM2X_SSPC_OAFC_HTSC:
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/AHU.yaml:5321]: Add ECON type

Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/AHU.yaml:5321]: Expect some sort of dehum control on this unit - need to determine exact sequence of ops for that

Comment (briancjohns) [reply ontology/yaml/resources/HVAC/entity_types/AHU.yaml]: Dehumidification uses zone air dewpoint compared to dewpoint setpoint.  If the zone is higher, chilled water valve is opened to 100% and the electric heating coil is modulated to reheat the air

File: ontology/yaml/resources/HVAC/entity_types/AHU.yaml [hunk 3]
```diff
@@ -5318,3 +5318,20 @@ AHU_BSPC_DX4SC_ECON_EFSS_EFVSC_SFSS_SFVSC_SSPC_RHHC_NONCANONICAL_1:
   - heating_water_valve_percentage_command
   - min_zone_air_relative_humidity_sensor
   - min_zone_air_relative_humidity_setpoint
+
+AHU_SFSS_SFVSC_EFSS_CHWSC_FDPM2X_SSPC_OAFC_ECON_ZDHC:
+  guid: "75559dbd-7ef6-4368-b48f-8d34db7fe47d"
+  description: "Multi zone AHU with exhaust fan, two prefilters, outside air, electric supply side heating, supply static pressure control and zone dehumidification"
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/AHU.yaml:5324]: If this is a multi-zone AHU, how does it control to a single zone air dewpoint sensor? Is it actually a return air dewpoint sensor?

File: ontology/yaml/resources/HVAC/entity_types/AHU.yaml (hunk 2 above)
Comment (cstirdivant) [reply ontology/yaml/resources/HVAC/entity_types/AHU.yaml]: How can a multi-zone AHU control to a single zone sensor? Is it actually the return air dewpoint sensor?

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 4]
```diff
@@ -7882,4 +7882,16 @@ DTPM:
...
+
+ZDHC:
+  guid: "d50c12f3-98bf-4da2-bc36-57d44160c789"
+  description: "zone dewpoint control with chilled water cooling and electric reheat"
+  is_abstract: true
+  uses:
+  - zone_air_dewpoint_temperature_sensor
+  - zone_air_dewpoint_temperature_setpoint
+  - chilled_water_valve_percentage_command
+  - heating_percentage_command
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:7895]: I would separate the heating control from here - you can probably use the existing HTSC type

**Review by cstirdivant — CHANGES_REQUESTED** [review]

**Review by shambergoldstein — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml (hunk 4 above)
Comment (BrianJohnsonOTI) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:7948]: No

File: ontology/yaml/resources/fields/telemetry_fields.yaml [hunk 5]
```diff
@@ -489,6 +489,9 @@ literals:
 - zone_air_dewpoint_temperature_sensor:
     flexible_min: 255.9259259
     flexible_max: 310.9259259
+- zone_air_dewpoint_temperature_setpoint:
```
[the remark this answers is not in the document]
Comment (BrianJohnsonOTI) [reply ontology/yaml/resources/fields/telemetry_fields.yaml:492]: No it doesn't

**Comment by shambergoldstein** [comment]
No longer needed, closing

**Pull request closed without merging**